### PR TITLE
APN Support

### PR DESCRIPTION
**Pull request by SkylerSeamans**

### CODE REVIEW

**Comment by SkylerSeamans** [comment]
Based off https://github.com/teamsnap/teamsnap-SDK-iOS/pull/128. Please work https://github.com/teamsnap/teamsnap-SDK-iOS/pull/128 first.

File: TeamSnapSDK/SDK/DataTypes/TSDKAPNDevices.m [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//
+
+#import "TSDKAPNDevices.h"
+
+@implementation TSDKApnDevices
+
+@dynamic token, appVersion, linkSelf, linkUser, linkRoot;
+
++ (NSString *)SDKType {
+    return @"apn_devices";
```
Comment (jrahaim) [inline TeamSnapSDK/SDK/DataTypes/TSDKAPNDevices.m]: I believe this should be apn_device for proper parsing.

**Comment by jrahaim** [comment]
I was getting a crash on [team copy] complaining that CollectionJSON didn't confirm to NSCopying. I did a quick implementation of it in 216a172 .

Other than that and it's 👍 for merge.

**Comment by jrahaim** [comment]
Needs a rebase too.

**Pull request merged**